pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -338,8 +322,7 @@ func (s *Server) handleCACertsFileWatch() {
 			}
 
 			if event.Op&fsnotify.Write == fsnotify.Write || event.Op&fsnotify.Create == fsnotify.Create {
-				valid := isValidCACertsFile(event.Name)
```

Comment:
another way is to watch files one by one, but we have 4 files to watch. So that's why we use dir watch here.

Reply:
I did not understand. What would be the `event.name` here?

Reply:
Take `/var/run/secrets/istio` directory as an examplem, it is mounted from istio rootcert configmap. 
When we update the configmap, the name can be `..data`, `..2022_04_13_09_13_52.574172247`

```
$ ls -al /var/run/secrets/istio
total 12
drwxrwxrwx 3 root root 4096 Apr 13 09:13 .
drwxr-xr-x 7 root root 4096 Apr 13 09:08 ..
drwxr-xr-x 2 root root 4096 Apr 13 09:13 ..2022_04_13_09_13_52.574172247
lrwxrwxrwx 1 root root   31 Apr 13 09:13 ..data -> ..2022_04_13_09_13_52.574172247
lrwxrwxrwx 1 root root   20 Apr 13 09:08 root-cert.pem -> ..data/root-cert.pem

```

Reply:
No event with name `root-cert.pem` come

Reply:
I see